cli/command/password/reset.go

```diff
@@ -24,6 +26,16 @@ func NewResetCommand(c cli.Interface) *cobra.Command {
 
 func reset(c cli.Interface, args []string) error {
 	conn := c.ClientConn()
+	clientVer := version.NewVersionClient(conn)
+	requestVer := &version.GetRequest{}
+	reply, err := clientVer.Get(context.Background(), requestVer)
+	if err != nil {
+		return fmt.Errorf("%s", grpc.ErrorDesc(err))
+	}
+	if reply.Info.Registration == "none" {
```

Comment:
Can you replace `"none"` by its constant definition `configuration.RegistrationNone` located in `cmd/amplifier/server/configuration/configuration.go`